### ISSUE TITLE
Can't import Seriously in a node environment 

### ISSUE THREAD

**Issue by juancabrera** [brianchirls/Seriously.js#91]
When I'm trying to load it:

```
import Seriously from '../libs/seriously';
```

Got this error on the browser console:
```
Uncaught TypeError: Cannot read property 'document' of undefined
```

It seems it's something related with the scope, not sure. When I modified it to export the main function it doesn't throw any error, but then got lots of other kind of errors because the library is not working properly after that big change (sure cuz I removed the main scope and export the main function directly)

Any ideas ?

**Comment by positlabs** [comment]
Check out #85

**Comment by juancabrera** [comment]
@positlabs :+1: 

that fixed it!

**Comment by brianchirls** [comment]
Thanks @juancabrera and @positlabs. I'll close this one for now as a duplicate and fix #85 as soon as I have time.
